src/types.rs

```diff
@@ -344,13 +345,29 @@ macro_rules! new_url_type {
             }
         }
         impl Eq for $name {}
+
+
+        #[cfg(feature = "schemars")]
+        impl schemars::JsonSchema for $name {
+            fn schema_name() -> String {
+                stringify!($name).to_owned()
+            }
+
+            fn schema_id() -> std::borrow::Cow<'static, str> {
+                std::borrow::Cow::Borrowed(concat!("oauth2::", stringify!($name)))
+            }
+
+            fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> schemars::schema::Schema {
+                // HELP(gibbz00): do we want to generate the schema for a URL or a String?
```

Comment:
I can see this question being relevant for the `openidconnect` PR, but in this crate they're all plain strings

Reply:
Oki, removing it.